data/scripts/Xsession

```diff
@@ -8,26 +8,11 @@
 # Note that the respective logout scripts are not sourced.
```

Comment:
I think this won't be true any more

Reply:
We `exec` at the end of script, I confirmed logout scripts have no possibility to run when we  `exec`.

Reply:
Oh true, never mind!